src/makeCalculator.js

```diff
@@ -37,7 +37,40 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
+    result: 0,
+
+    add(x) {
+      obj.result += x;
+    },
+
+    subtract(x) {
+      obj.result -= x;
+    },
+
+    multiply(x) {
+      obj.result *= x;
+    },
+
+    divide(x) {
+      obj.result /= x;
+    },
+
+    reset() {
+      obj.result = 0;
+
+      return obj;
+    },
+
+    operate(x, y) {
```

Comment:
How the calculator will work:
 * - Each `operate` call takes a callback and a number and **sets the
 *   appropriate value to the `result` property.**
in your case the result property is changing in methods: `add`, `subtract`, `multiply`, `divide`, but not in 'operate'